### ISSUE TITLE
Problems with scrolling in the logger

### ISSUE THREAD

**Issue by dimisa-RUAdList** [uBlockOrigin/uBlock-issues#2124]
### Prerequisites

- [X] I verified that this is not a filter issue (MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues))
- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue
- [X] The issue is not present after wholly disabling uBlock Origin ("uBO") in the browser
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### Description

When using a logger, scrolling stops working after shifting a few lines

### A specific URL where the issue occurs

Any address, for example: https://3dnews.ru/

### Steps to Reproduce

1. Open `https://3dnews.ru/`
2. Open logger
3. Click the refresh button to populate the logger
4. Use the mouse scroll wheel or the up/down buttons

### Expected behavior

Scrolling works as usual, the lines are shifted to a given position

### Actual behavior

Scrolling stops working after shifting a few lines

### uBlock Origin version

uBlock Origin development build 1.42.5rc0, uBlock Origin 1.42.4

### Browser name and version

Google Chrome 102.0.5005.63 (win), Google Chrome mac 102.0.5005.61 (mac)

### Operating System and version

Windows 7, macOS Monterey

**Comment by Yuki2718** [comment]
I can't reproduce on Windows, scrolling works fine after clicking the scroll button (rotating arrows).

**Comment by gwarser** [comment]
102.0.5005.61 1.42.4

I can reproduce. It's not like it completely stops, but it feels like a few seconds long freeze. Like broken, skipping scroll. But it's fine in Firefox.

**Comment by dimisa-RUAdList** [comment]
That's right. If you move the cursor, the scrolling starts working again, but after scrolling several lines, the stop repeats.

**Comment by gwarser** [comment]
I have other issues with input in Chrome. For example, context menu randomly does not appear. Maybe broken Chrome build?

**Comment by hirorpt** [comment]
Maybe same issue?
https://discuss.codemirror.net/t/scrolling-issues-on-latest-google-chrome/4444

**Comment by dimisa-RUAdList** [comment]
Initially, I thought that this problem was observed only in uBO, but now I have already checked other extensions. As it turned out, this problem applies to all extensions, that is, it is related to the browser itself. For example, if open the script code editor in Tampermonkey, then scrolling behaves in a similar way.
